### ISSUE TITLE
Can't move cursor using keyboard in the user status message popup

### ISSUE THREAD

**Issue by Linerly** [vector-im/element-web#19881]
### Steps to reproduce

1. Enable custom user status messages in Labs
2. Write something in the status section (click profile picture in composer), then try to use the arrow keys to move the cursor or use the Home/End keys to go to the start/end of the line

#### What did you expect?
Be able to move the cursor by using the arrow keys and go to start/end of the line using the Home/End keys

#### What happened instead?
Trying to move the cursor using the arrow keys closes the status popup
Using the Home/End keys doesn't work as well

### Operating system

Arch Linux

### Application version

Element Nightly version: 2021111601 Olm version: 3.2.3

### How did you install the app?

AUR

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by SimonBrandner** [comment]
FYI, IIRC, the popup is getting replaced soon

**Comment by Linerly** [comment]
This is still happening with the new UI
